bulletin_board/server/Gemfile

```diff
@@ -23,7 +23,7 @@ gem "turbolinks", "~> 5"
 # Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
 gem "jbuilder", "~> 2.7"
 # Use Redis adapter to run Action Cable in production
-gem "redis", "~> 4.0"
+# gem "redis", "~> 4.0"
```

Comment:
Why are we removing `redis`? 👓

Reply:
Nope, but I think we are not using that gem anymore, as we don't use Action Cable. Redis will be used when using Sidekiq for ActiveJobs, but currently is not configured.